admin_guide/managing_networking.adoc

```diff
@@ -144,6 +144,36 @@ $ systemctl restart atomic-openshift-master-api atomic-openshift-master-controll
 
 ----
 
+[[admin-guide-managing-networking-ingress]]
+== Setting up the HAProxy router to accept ingress objects
+
+By default, the HAProxy router only accepts route objects. You can set up the
+HAProxy router as a Kubernetes
+link:https://kubernetes.io/docs/concepts/services-networking/ingress/[Ingress
+Controller] so that it accepts both route and ingress objects. If configured,
+the same rules apply for claiming a domain for a namespace based on the creation
+time of the object.
+
+While this makes the router compatible with Kubernetes ingress
+objects, some caveats exist:
+
+* Ingress objects store the keys and certificates in secrets, so the router needs permission to read all secrets in the system.
+* Ingress objects only support edge termination for `https` routes.
+
+To configure an existing router to have Ingress support (assuming the default name of `router` for the deployment configuration and the ):
```

Comment:
This sentence trailed off. Do you remember what was supposed to be here?

Reply:
Oops.  "and the service-account"